pro_tes/ga4gh/tes/task_runs.py

```diff
@@ -529,7 +555,16 @@ def _update_doc_in_db(
     def _update_task_metadata(
         self, db_document: DbDocument, tes_uri: str, remote_task_id: str
     ) -> DbDocument:
-        """Update the task metadata."""
+        """Update the task metadata.
```

Comment:
Again, more details would be nice. How does this differ from the previous function? Isn't all the data metadata, really?

Try to find a succinct (not more than one line!) way of summarizing the function. Then, if you feel that you need to describe more, you can add an extended description (leave one line blank after the summary, and before the "Args:" section).

Reply:
Done

Reply:
Yup. But please add a `.` at the end of the description. Also, it would be good to put backticks around any expressions like `TesTask.logs.metadta`.

Reply:
Fixed

Reply:
And the backticks? Put:

```console
[...] expressions like `TesTask.logs.metadata`.
```

Reply:
Done